springboot3-todo/Makefile

```diff
@@ -0,0 +1,29 @@
+UNAME := $(shell uname)
+
+ifeq ($(UNAME), Linux)
+	os=linux
+endif
+ifeq ($(UNAME), Darwin)
+	os=mac
+endif
+
+.PHONY: build build-mac build-linux clean
+
+build: build-$(os)
+
+build-mac: target/todo.jar 
+	docker build -q -f src/main/docker/Dockerfile -t spring/todo .
+
```

Comment:
we should tag this as `spring3/todo` or something to distinguish it from the spring2 image

Reply:
I wasn't sure what the purpose of the `Makefile` in there was...if it was even used anymore for anything (seems wrong/inaccurate for the other projects in there too).

i copied the existing spring folder in the repo to the new one and updated it. But you're right, if it is used then we should update it.

Reply:
You are correct, they haven't been maintained, lets leave it for now